### ISSUE TITLE
[FEATURE] Add flags to control verbosity of output

### ISSUE THREAD

**Issue by shonfeder** [informalsystems/apalache#278]
When a user isn't concerned with the timestamps that indicate how long steps take to run, this part of the output can add noisy and non-deterministic output that doesn't add value. It could be useful to add a way to silence this when desired.

This would be useful, for instance, in cleaning up our integration tests.

See https://github.com/informalsystems/apalache/pull/270#discussion_r503522422 for context.

**Comment by konnov** [comment]
The unix tools are usually able to detect whether they output to the terminal. In that case, I would leave the time stamp and remove them, when the output is different from tty. However, I have no idea about how to detect that in JVM.

**Comment by shonfeder** [comment]
Oooks like this could help: https://stackoverflow.com/questions/1403772/how-can-i-check-if-a-java-programs-input-output-streams-are-connected-to-a-term

I still doubt whether the time stamps on stdout on every line are useful at all, or indeed, whether most of the output is really useful (especially with the detailed.log readily available) ;)

**Comment by konnov** [comment]
Colored output on the terminal would be awesome :-)

**Comment by konnov** [comment]
We can always add the quiet flag `-q`. That should be easy.

**Comment by shonfeder** [comment]
At https://github.com/informalsystems/apalache/pull/1279#pullrequestreview-870601372 Igor suggested adjusting whether the timestamps are printed based on whether we're using a TTY.

**Comment by konnov** [comment]
I am also getting annoyed by the timestamps from time to time. How about the following rather stupid solution: Remove the timestamps and write a wrapper script for `apalache-mc` that adds timestamps to every line of stdout? This can be probably done inside the shell script too.

**Comment by konnov** [comment]
Obviously, someone has thought about this too: https://serverfault.com/questions/310098/how-to-add-a-timestamp-to-bash-script-log

**Comment by shonfeder** [comment]
Why not just make this configurable in the Scala logging? Why would we move it into a wrapper script?
